src/main/java/org/apache/sysml/utils/Statistics.java

```diff
@@ -621,6 +636,8 @@ public static String display(int maxHeavyHitters)
 		//show extended caching/compilation statistics
 		if( DMLScript.STATISTICS ) 
 		{
+			String blas = NativeHelper.blasType != null ? NativeHelper.blasType : ""; 
+			sb.append("Native " + blas + " calls (Mult/DNN/Fails):\t" + numNativeLibMatrixMultCalls.get()  + "/" + numNativeLibMatrixDNNCalls.get() + "/" + numNativeFailures.get() + ".\n");
```

Comment:
please only show this line if there was at least one call to native blas libraries.